com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs

```diff
@@ -21,14 +21,14 @@ protected override bool DisableNameSelection
         {
             get
             {
-                return (GetToExport() != null && GetToExport().Length > 1);
+                return (ToExport != null && ToExport.Length > 1);
```

Comment:
Is this coming from a merge? It isn't bad exactly, just seems like noise.

But if we are changing things, here we could store ToExport in a local variable so we only call the getter once.

Reply:
no, I just changed it from functions to a property

Reply:
Not a big deal but I'd recommend doing that in its own PR.